t

doctorwwho opened the pull request.

minglich committed:
Update adv_mode.lua

minglich committed:
AK

minglich committed:
Update adv_mode.lua

cause I use other script to test recoil , so maked some error recently . this script can use now , if the recoil compensation is large or small ， you can modify the value of all_recoil_Times or recoil_table{times} to fix . based recoil is test on gun with no parts , full recoil is equip compensator and grip  ,  but I haven't pick up these ,  so full mode is empty now.

minglich committed:
Add holdbreath and other things

When your numberlock is on  , the recoil is full_mode ,  with full attachments . When you press left shift,the recoil will times recoil_table{holdbreathtimes} , I have only test ump9 .

minglich committed:
Update README.md

minglich committed:
add auto_mode

just recoil except m16.

minglich committed:
Add files via upload

sensitivity in game

minglich committed:
Update README.md

minglich committed:
Update adv_mode.lua

circle the last value in recoil table.

minglich committed:
Update README.md

minglich committed:
Update adv_mode.lua

minglich committed:
Update adv_mode.lua

start script,scrolllock lamp will be on.close script ,scrolllock lamp will be off.

confeter commented:
Hi,
can you tell me if i can use it with 1.00 vertical sensitivity instead of 0.70 ?
Can you add a thing that stop rapid fire after 30/40 bullets, because you can hear the rapidfire when the mag is empty and could be reported also

thanks

minglich committed:
Update adv_mode.lua

support logitech g keyboard and you can close some recoil now.

minglich committed:
ump

confeter commented:
Can you share your table for 1.0 vertical sensitivity ?

minglich committed:
add vertical_sensitivity and fix something.

busy in work recently , no time to play pubg and test recoil..pls help me to test the full-mode and 4x-mode ,if you have the correct values of Times , send to me ,thks.

minglich committed:
Update adv_mode.lua

minglich committed:
Add fastloot

when you press fast_loot_key and click Left mouse button，script will start fastloot untill release fast_loot_key ( default is left ctrl )

minglich committed:
Update README.md

minglich committed:
Update README.md

minglich committed:
Fix M16

confeter commented:
Hi,

can i use it without logitech mouse if i have an logitech keyboard?

minglich committed:
Add files via upload

minglich committed:
Add files via upload

minglich committed:
Update README.md

minglich committed:
Fix Bugs and make fastloot better for work.

minglich committed:
Update adv_mode.lua

minglich committed:
test

test

minglich committed:
Update test.lua

minglich committed:
Update test.lua

minglich committed:
Update and rename test.lua to weaponspeed_mode.lua

make the script more fluent ，add auto-reload，fix some recoil.

minglich committed:
Update README.md

minglich committed:
Update weaponspeed_mode.lua

minglich committed:
Update weaponspeed_mode.lua

minglich committed:
remove fire_key 

no need to change the fire key now.

minglich committed:
Update README.md

minglich committed:
Update README.md

minglich committed:
Update weaponspeed_mode.lua

minglich committed:
Update weaponspeed_mode.lua

minglich committed:
Update weaponspeed_mode.lua

minglich committed:
Update weaponspeed_mode.lua

minglich committed:
Update weaponspeed_mode.lua
